### PR TITLE
add horoscope feature

### PR DESCRIPTION
**Pull request by paveethan**
To my current understanding, this needs to be accomodated in api.api.ai before it can be tested, since during testing it is not recognizing this as an intent.

### CODE REVIEW

**Comment by gauthamzz** [comment]
what are the phrases you want to initiate this action. Suggest more than one.

Presently i added the feature and horoscope for aries' should work

File: background.js [hunk 1]
```diff
@@ -712,6 +714,22 @@ function swapTab() {
       });
   }
 
+  function getHoroscope(sign) {
+      var url = "http://horoscope-api.herokuapp.com/horoscope/today/"+sign;
+      var linkUrl = "https://www.ganeshaspeaks.com/horoscopes/daily-horoscope/"+sign;
+      var responseText = "Today's horoscope for "+sign+", ";
+     $.getJSON(quoteUrl, function(data) {
+          setResponse(responseText+data.horoscope);
```
Comment (gauthamzz) [inline background.js]: This looks good

**Comment by paveethan** [comment]
get horoscope for aries/leo/sagittarius/taurus/virgo/capricorn/gemini/libra/aquarius/cancer/scorpio/pisces
for the time being.

**Review by gauthamzz — APPROVED** [review]

**Review by gauthamzz — APPROVED** [review]

**Comment by gauthamzz** [comment]
It should be working now, can you check :smile:

**Comment by gauthamzz** [comment]
I am getting this error
deepinscreenshot_select-area_20180314023910

File: background.js [hunk 2]
```diff
@@ -712,6 +714,22 @@ function swapTab() {
...
+          chrome.tabs.create({
+              'url': linkUrl
```
Comment (gauthamzz) [inline background.js]: this should be url i guess

**Review by gauthamzz — CHANGES_REQUESTED** [review]
background.js:721 Uncaught ReferenceError: quoteUrl is not defined
    at getHoroscope (background.js:721)
    at Object.success (background.js:399)
    at j (jquery.min.js:2)
    at Object.fireWith [as resolveWith] (jquery.min.js:2)
    at x (jquery.min.js:4)
    at XMLHttpRequest.<anonymous> (jquery.min.js:4)
getHoroscope @ background.js:721
success @ background.js:399
j @ jquery.min.js:2
fireWith @ jquery.min.js:2
x @ jquery.min.js:4
(anonymous) @ jquery.min.js:4
XMLHttpRequest.send (async)
send @ jquery.min.js:4
ajax @ jquery.min.js:4
tasks @ background.js:278
send @ background.js:270
setInput @ background.js:252
recognition.onend @ background.js:180

Please fix this error

**Comment by paveethan** [comment]
53982ea35b5897ea1b80bf74282144ce232c402a 

Yep was building on top of another method and forgot to switch url

**Comment by gauthamzz** [comment]
This works i checked. but is this right 

```
Unchecked runtime.lastError while running tabs.executeScript: The tab was closed.
    at Object.callback (chrome-extension://bndgleobglnnnngbkeilepjdhnnhlloc/background.js:118:37)
jquery.min.js:4 GET http://horoscope-api.herokuapp.com/horoscope/today/ 404 (Not Found)
```

File: background.js [hunk 3]
```diff
@@ -712,6 +714,22 @@ function swapTab() {
...
   }
 
+  function getHoroscope(sign) {
+      var url = "http://horoscope-api.herokuapp.com/horoscope/today/"+sign;
+      var linkUrl = "https://www.ganeshaspeaks.com/horoscopes/daily-horoscope/"+sign;
+      var responseText = "Today's horoscope for "+sign+", ";
+     $.getJSON(url, function(data) {
+          setResponse(responseText+data.horoscope);
+          chrome.tabs.create({
+              'url': linkUrl
```
Comment (gauthamzz) [inline background.js:724]: I think this should be 'url' :url

Its always opening the ganesha speaks website

**Comment by paveethan** [comment]
That error is occuring because 'sign' is not being passed along, I'll look
into the issue now.

**Comment by paveethanr** [comment]
I've tested it and it seems to be working fine on my end; and that is the
page that is supposed to open. The other URL is just the JSON file, and the
ganesha speaks website has more information, so users would find that more
appealing as opposed to seeing the json file.

File: background.js (hunk 3 above)
Comment (paveethan) [reply background.js:724]: I've tested it and it seems to be working fine on my end; and that is the
page that is supposed to open. The other URL is just the JSON file, and the
ganesha speaks website has more information, so users would find that more
appealing as opposed to seeing the json file.

**Comment by paveethan** [comment]
The only other issue remaining is that there is a character limit of 256,
anything more and api.ai will not parse it, a decision needs to be made
whether Anna responds with a truncated message or responds with a generic
message "here is your horoscope for today". Here is an example of it
working: https://streamable.com/awc4w

**Comment by gauthamzz** [comment]
thats alright for now. Try to make the error go. Everything else LGTM

**Comment by paveethan** [comment]
Which error? I submitted a bugfix for the url error.

**Comment by gauthamzz** [comment]
Oh my mistake :small_airplane: 
Thanks for this

**Comment by gauthamzz** [comment]
If you can add this in the docs, it would be great :smile:

**Pull request merged**